packages/block-editor/src/utils/transform-styles/transforms/url-rewrite.js

```diff
@@ -1,8 +1,3 @@
-/**
- * External dependencies
- */
-import { parse, resolve } from 'url';
```

Comment:
Should it also get removed from the `package.json` file or was it never listed?

Reply:
I double-checked and I guess ESLint rule ignores Node dependencies.

Reply:
> Should it also get removed from the `package.json` file or was it never listed?

It was never included. It's a bit of a "special" module, discussed also at #13386, where Webpack will provide shims for this and other Node built-ins. Technically it probably was never necessary to explicitly include it in `block-library/package.json`.

See: https://webpack.js.org/configuration/node/#other-node-core-libraries